socs/agents/hwp_encoder/agent.py

```diff
@@ -695,6 +726,7 @@ def make_parser(parser=None):
     # Add options specific to this agent.
     pgroup = parser.add_argument_group('Agent Options')
     pgroup.add_argument('--port', type=int, default=8080)
+    pgroup.add_argument('--ip', type=str, default='None')
```

Comment:
At minimum we need some description here. With the only options being `--port` and `--ip` one could easily assume this is the IP and port on the beaglebone, when really it's a listening port in the Agent for receiving UDP packets, and the IP of the beaglebone.